lib/ruby_lsp/ruby_lsp_rails/definition.rb

```diff
@@ -88,7 +88,8 @@ def handle_route(node)
         result = @client.route_location(T.must(node.message))
         return unless result
 
-        file_path, line = result.fetch(:location).split(":")
+        *file_parts, line = result.fetch(:location).split(":")
+        file_path = file_parts.join(":")
```

Comment:
Might be worth a comment to highlight that approach is needed because of Windows.